File: jambeth/jambeth-merge/src/main/java/de/osthus/ambeth/util/setup/IDataSetup.java
package de.osthus.ambeth.util.setup;

import java.util.Collection;

public interface IDataSetup
{
	void eraseEntityReferences();

	void refreshEntityReferences();

	IDataSetupWithAuthorization resolveDataSetupWithAuthorization();

	Collection<Object> executeDatasetBuilders();
}

File: ambeth/Ambeth.Merge/ambeth/util/setup/IDataSetup.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace De.Osthus.Ambeth.ambeth.util.setup
{
    public interface IDataSetup
    {
        IList<Object> ExecuteDatasetBuilders();
    }
}
